rest-api-spec/src/main/resources/rest-api-spec/api/fleet.global_checkpoints.json

```diff
@@ -0,0 +1,47 @@
+{
+  "fleet.global_checkpoints":{
+    "documentation":{
+      "url": null,
+      "description": "Returns the current global checkpoints for an index."
```

Comment:
Can we add to the description that this is intended to be consumed by fleet-server only?